app/views/books/index.html.erb

```diff
@@ -1,35 +1,50 @@
 <p id="notice"><%= notice %></p>
 
-<h1>Books</h1>
+<div class="container-fluid">
+  <h4>Books</h4>
```

Comment:
On the semantic HTML thing, this should probably still be an h1, since it's the main title of the page. If that comes out too large, fix it in CSS, rather than using an h4.